pkg/chain/bitcoin/handle.go

```diff
@@ -0,0 +1,8 @@
+package bitcoin
+
+// Handle serves as an interface abstraction around bitcoin network queries
+type Handle interface {
+	Broadcast(transaction string) error
```

Comment:
Side note: Would it be worth returning a transaction ID from the `Broadcast` function?

Reply:
I think that has the potential to be useful, but since we wouldn't be doing anything with it at the moment, I'd prefer to leave it out until we have a concrete use case

Reply:
We could log the ID on the caller level instead of doing so inside the implementation.